components/nds-card/src/Card.js

```diff
@@ -47,7 +50,9 @@ const CardBody = props => {
 						return (
 							<div key={`item${idx}`} className="card__metadatum">
 								{item.label && (
-									<dt className="visually-hidden">{item.label}</dt>
+									<dt className={!item.visibleLabel && "visually-hidden"}>
```

Comment:
This is causing this issue:
![image](https://user-images.githubusercontent.com/893676/125844777-95f291e7-32fc-4002-bb3e-dfb5932d0611.png)